plugins/soundsourcem4a/soundsourcem4a.cpp

```diff
@@ -126,6 +126,11 @@ Result SoundSourceM4A::tryOpen(const AudioSourceConfig& audioSrcCfg) {
         return ERR;
     }
 
+    if (MP4GetTrackFixedSampleDuration(m_hFile, m_trackId) != kFramesPerSampleBlock) {
```

Comment:
add a comment:

// TODO(lp:#XXXXX): Remove this error condition when we support this type of file